client/my-sites/patterns/components/header/index.tsx

```diff
@@ -3,20 +3,29 @@ import Search from 'calypso/components/search';
 import './style.scss';
 
 type PatternsHeaderProps = {
-	title: string;
 	description: string;
+	initialSearchTerm?: string;
+	onSearch?( searchTerm: string ): void;
+	title: string;
 };
 
-export const PatternsHeader = ( { title, description }: PatternsHeaderProps ) => {
+export const PatternsHeader = ( {
+	description,
+	initialSearchTerm = '',
+	onSearch = ( s ) => s,
```

Comment:
I'm a bit lost. According to the type definition, this function should return nothing. Am I missing something?

```suggestion
	onSearch = () => {},
```

Reply:
> Am I missing something?

Quite the opposite! 👍